toit/benchmark.toit

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2020 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
```

Comment:
Add LICENSE file somewhere and update these headers.

Reply:
done.